uefi/flash.go

```diff
@@ -84,6 +109,88 @@ func (f *FlashImage) Validate() []error {
 	return errors
 }
 
+// Extract extracts the flash image to the directory passed in.
+func (f *FlashImage) Extract(dirPath string) error {
```

Comment:
same feedback as above about collapsing error handling lines and using `os.Create`

Reply:
will do

Reply:
collapsed the error handling.